src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/ReferenceConfidenceModel.java

```diff
@@ -555,7 +555,8 @@ int sumMismatchingQualities(final byte[] readBases,
         for ( int i = 0; i < n; i++ ) {
             final byte readBase = readBases[readStart + i];
             final byte refBase  = refBases[refStart + i];
-            if ( !Nucleotide.decode(readBase).intersect(Nucleotide.decode(refBase)).isValid()) {  //allow N read bases (and other ambiguous bases) to "match" reference
+            if ( !Nucleotide.decode(readBase).intersect(Nucleotide.decode(refBase)).isValid() &&
```

Comment:
Ah I see, I was assume that master had a change that I have on Nucleotides adding an new intersect(byte, byte) method:
```
/**
 * Checks whether two nucleotides intersect given their byte encodings.
 * @param a first nucleotide.
 * @param b second nucleotide.
 * @return {@code true} iff the input nucleotides intersect.
 */
public static boolean intersect(final byte a, final byte b) {
  return (baseToValue[0xFF & a].mask & baseToValue[0xFF & b].mask) != 0;
}
```

Just add it in your branch's Nucleotides.java and the the if code would be:

```
if (!Nucleotide.intersect(readBase, refBase) && readBase != AlignmentUtils.PLACEHOLDER) {
   ....
}
```